deployment_report/model/viya_deployment_report.py

```diff
@@ -243,14 +243,11 @@ def gather_details(self, kubectl: KubectlInterface,
                                                            sas_custom_resource_types=sas_custom_resource_types)
 
         #######################################################################
-        # Get configuration values from ConfigMaps                            #
+        # Create a cadence/db information                                     #
         #######################################################################
-        # get the ConfigMap resources that were cached above
-        config_maps: Dict = resource_cache[ResourceTypeValues.K8S_CORE_CONFIG_MAPS][ITEMS_KEY]
+        cadence_info: Optional[Text] = config_util.get_cadence_version(resource_cache)
```

Comment:
Best practice is to reference the param by name:

```python
config_util.get_cadence_version(resource_cache=resource_cache)
```

That way, if the signature of that method were to change in the future we would not need to adjust the order of params or miss the change and produce unexpected results.